### ISSUE TITLE
float is not working for span in td 

### ISSUE THREAD

**Issue by laxman1223** [dompdf/dompdf#1299]
```
<!doctype html>
<html>
<head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
    <title>Admit Card</title>
 <style>
 tr.border td {
  border:2pt solid black;
  padding:10px;
  border-radius: 5px;
}
table
{
	border-collapse:separate; 
    border-spacing:10px;
}
</style> 
</head>

<body>
    <div style="font-family: 'Open Sans', sans-serif;">
        <table  style="width:100%;">
                        <tr style="width:100%;">
                            <td style="width:50%;">
                                <img src="#" style="width:300px;">								
                            </td>                         
                            <td align="right" >
                                 <img src="#" style="width:300px;height:100px;">
                            </td>
                        </tr>
			<tr  style="width:100%;">                                              
                            <td  colspan="2"  align="center" >
                                 <b style="font-size:24px;color:#00000;" >Admitcard</b>
                            </td>
                        </tr>
			<tr  class="border" style="width:100%;font-size:12px;">
                            <td colspan="2"  >
                                <span style="float:left;" ><b>C.No:</b>CAM000028</span>								
                                 <span style="float:right;" ><b>Date:</b>27/10/2016</span>
                            </td>													
                        </tr>
        </table>
</div>
</body>
</html>
```

**Comment by laxman1223** [comment]
[Fri Nov 04 16:41:07.373350 2016] [:error] [pid 5288:tid 1680] [client ::1:51386] PHP Notice:  Array to string conversion in C:\\xampp\\htdocs\\cm\\vendor\\dompdf\\dompdf\\src\\Css\\Style.php on line 2569,

**Comment by bsweeney** [comment]
Is your code sample correct? The image tag is a bit off:

```
<img src="#"; ?>" style="width:300px;height:100px;">
```

**Comment by laxman1223** [comment]
Check update code now...

**Comment by bsweeney** [comment]
Should be addressed in 0.7.1 (fix is currently in the develop branch).
